accumulator_updater/Anchor.toml

```diff
@@ -4,6 +4,8 @@ skip-lint = false
 [programs.localnet]
 accumulator_updater = "Fg6PaFpoGXkYsidMpWTK6W2BeZ7FEfcYkg476zPFsLnS"
 mock_cpi_caller = "Dg5PaFpoGXkYsidMpWTK6W2BeZ7FEfcYkg476zPFsLnS"
+#accumulator_updater = "3xzsrbvWEM4SMM2W7wbVofpDi3zrdhb2Z2xLwMEGJQiZ"
+#mock_cpi_caller = "EKwG38D6Sd67NqsAVc4R7Hqr6mZFPzFotzBchoQJRRDe"
```

Comment:
why is this commented out? (delete or leave a comment as to why you would want to uncomment)